src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
I find `propEq` very useful. I also find `eqProps` useful. I never use `propIs` but that does not give me an urge to remove it from the code base. If others use it, that is fine by me. Same goes for `R.T` and `R.F`.